controllers/upgrade_handlers.go

```diff
@@ -24,15 +24,16 @@ import (
 	"strings"
 	"time"
 
+	lcautils "github.com/openshift-kni/lifecycle-agent/utils"
```

Comment:
should be part if next imports block

Reply:
What do you mean? btw it was auto sorted and moved at the top. (L35 now L27)